File: exs/mundo_1/javascript/006.js
/*
Desafio 006

Problema: Crie um algoritmo que leia um número e mostre o seu dobro, o seu triplo e sua raiz quadrada

Resolução do problema:
*/

const input = require('readline-sync').question

let num = Number(input('Insira um número inteiro: '))
let dobro = num*2
let triplo = num*3
let raizQuadrada = Math.sqrt(num)

console.log(`O dobro de ${num} é ${dobro}`)
console.log(`O triplo é ${triplo}`)
console.log(`E a raiz quadrada é ${raizQuadrada.toFixed(2)}`)

File: exs/mundo_1/golang/006.go
/*
	Desafio: 006
	Problema: Crie um algoritmo que leia um número 
	e mostre o seu dobro, o seu triplo 
	e sua raiz quadrada.
*/

package main 
import (
	"fmt"
	"math"
)

func main(){
	var number float64
	var double float64
	var triple float64
	var root float64
	fmt.Print("Informe um número: ")
	fmt.Scan(&number)
	double = number * 2
	triple = number * 3
	root = math.Sqrt(number)
	fmt.Println("Dobro:", double)
	fmt.Println("Triplo:", triple)
	fmt.Println("Raiz:", root)
}